src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
So my master was updated fully, so I just did a git merge master with my cholesky_improvement.  This seems to have everything updating.